packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -45,45 +45,72 @@ describe('SendTokenToAddress', () => {
     await container.generate(1)
   }
 
+  async function sendTokensToAddress (from: AddressBalances, to: AddressBalances, selectionMode?: SelectionModeType): Promise<string> {
+    const transactionHex = await client.account.sendTokensToAddress(from, to, selectionMode)
+    await container.generate(1)
+    return transactionHex
+  }
+
   it('should create a transaction with auto select (empty source address)', async () => {
-    const transactionHex = await client.account.sendTokensToAddress({}, { [await client.wallet.getNewAddress()]: ['2@DFI'] })
+    const address = await client.wallet.getNewAddress()
+    const transactionHex = await sendTokensToAddress({}, { [address]: ['2@DFI'] })
```

Comment:
line 49 is correct.

Reply:
<img width="1042" alt="Screenshot 2021-06-07 at 4 16 22 PM" src="https://user-images.githubusercontent.com/7101806/120982855-b9794680-c7ab-11eb-8541-37aa26054663.png">

Reply:
![image](https://user-images.githubusercontent.com/57817584/120982974-2ae7d380-c771-11eb-80bc-d01949fcbf0b.png)

Had to change it because of line 66

Reply:
No need this line -> `const address = await client.wallet.getNewAddress()`

Reply:
Write like this:
`const transactionHex = await sendTokensToAddress({}, { [await client.wallet.getNewAddress()]: ['2@DFI'] })`

Reply:
Okay, my fault. Sorry.

Reply:
okay